apps/admin-gui/src/_styles.scss

```
@@ -40,21 +45,22 @@ $perun-accent: mat.define-palette(mat.$green-palette, 600);
 $perun-warn: mat.define-palette(mat.$red-palette);
 
 // Create the theme object (a Sass map containing all of the palettes).
-//FIXME(theme-error): old definition of theme - will be able to be removed if the new definition is correct
-// $perun-theme: mat.define-light-theme($perun-primary, $perun-accent, $perun-warn);
 $perun-theme: mat.define-light-theme(
   (
     color: (
       primary: $perun-primary,
       accent: $perun-accent,
       warn: $perun-warn,
     ),
+    typography: mat.define-typography-config(),
+    density: 0,
   )
 );
 
 // Include theme styles for core and each component used in your app.
 // Alternatively, you can import and @include the theme mixins for each component
 // that you are using.
+$mat-theme-ignore-duplication-warnings: true;
```

```suggestion
mat.$theme-ignore-duplication-warnings: true;
```
it works when I change it to this.

---

Fixed.